src/main/java/com/deusgmbh/xcusatio/api/services/APIServicesBaseTest.java

```diff
@@ -0,0 +1,52 @@
+package com.deusgmbh.xcusatio.api.services;
+
+import com.deusgmbh.xcusatio.api.APIService;
+import com.deusgmbh.xcusatio.data.usersettings.UserSettings;
+
+public class APIServicesBaseTest extends APIService {
+
+    public static void main(String[] platinum) {
+        APIService apiService = new APIService() {
+
+            @Override
+            public void transmitDataToWebsite() {
+                // TODO Auto-generated method stub

```

Comment:
Kommt da noch was hin? Falls ja in das TODO reinschreiben, was noch gemacht werden muss, damit derjenige, der das dann macht, daran denkt.

Reply:
Denke das passt schon. Ist ja Jan der das selbst implementieren muss. Das sind die abstrakten methoden die jeder APIService implementieren muss und daher sind die schon alle da